lib/annotations/hooks/AfterBulkCreate.ts

```diff
@@ -0,0 +1,21 @@
+import {IHookOptions} from "../../interfaces/IHookOptions";
+import {addHook} from "../../services/hooks";
+
+export function AfterBulkCreate(target: any, propertyName: string): void;
+export function AfterBulkCreate(options: IHookOptions): Function;
+export function AfterBulkCreate(...args: any[]): void|Function {
+
+  if (args.length === 1) {
```

Comment:
I would suggest to separate this (line 8-20) to an additional method in the hooks service cause these lines appear in every hook annotation. What do you think?